### PR TITLE
Add test for detect_scenes

### PR DESCRIPTION
**Pull request by scherroman**

### CODE REVIEW

File: tests/videos/tools/test_cuts.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from moviepy.editor import ColorClip, concatenate_videoclips
+from moviepy.video.tools.cuts import detect_scenes
+
+
+def test_detect_scenes__red_green():
```
Comment (mbeacom) [inline tests/videos/tools/test_cuts.py]: Any reason for the double `__` ?

File: tests/videos/tools/test_cuts.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    """
+    Test that a cut is detected between concatenated red and green clips
+    """
+    red = ColorClip((640, 480), color=(255, 0, 0)).set_duration(1)
+    green = ColorClip((640, 480), color=(0, 200, 0)).set_duration(1)
+    video = concatenate_videoclips([red, green])
+
+    cuts, luminosities = detect_scenes(video, fps=10, progress_bar=False)
+
+    assert len(cuts) == 2
```
Comment (mbeacom) [inline tests/videos/tools/test_cuts.py]: Can you add a new line to the end of this file?

File: tests/videos/tools/test_cuts.py (hunk 1 above)
Comment (scherroman) [inline tests/videos/tools/test_cuts.py]: I use double underscores to separate the function being tested from the type of test, to help readability

File: tests/videos/tools/test_cuts.py (hunk 2 above)
Comment (scherroman) [inline tests/videos/tools/test_cuts.py]: Done

**Pull request merged**